gateway-api/docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM knotx/knotx:2.0-RC3
```

Comment:
If we decide to use `master` branch approach from the previous comment we would not be able to use "released" Docker image.